manager/Makefile

```diff
@@ -21,7 +21,7 @@ build-linux: generate vet
 
 # Overwrite docker-build from docker.mk
 docker-build: generate
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -o manager main.go
 	docker build . -t ${IMG} -f Dockerfile
 
```

Comment:
No cleanup of `manager` binary here? Alternatively we can avoid those cleanups and always build to ROOTDIR/bin that will be added to .gitignore.